wasm/src/main/java/com/dylibso/chicory/wasm/types/NameCustomSection.java

```diff
@@ -26,7 +26,7 @@ public class NameCustomSection extends CustomSection {
     /**
      * Construct a section instance from the specified contents.
      */
-    public NameCustomSection(
+    private NameCustomSection(
```

Comment:
We should remove all of the `add` methods and ideally make immutable copies of the lists in the constructor.

Reply:
done